bibliopixel/drivers/serial_driver.py

```diff
@@ -220,12 +224,12 @@ def _connect(self):
             raise e
 
     @staticmethod
-    def setDeviceID(dev, id):
+    def setDeviceID(dev, id, baudrate=921600):
```

Comment:
Eventually, these could just be pure functions and not static methods, but that's for another day.  :-)